### ISSUE TITLE
[Problem/Bug]: On Share , new window is opened , but event not coming on newWindowRequested

### ISSUE THREAD

**Issue by rahullli** [MicrosoftEdge/WebView2Feedback#4938]
### What happened?

<img width="303" alt="image" src="https://github.com/user-attachments/assets/cb15bd63-0e3f-4fba-ba2f-bbeef30b483d">
<img width="859" alt="image" src="https://github.com/user-attachments/assets/69d29751-fa36-4c1b-8d60-3ed96578e436">

I am trying to share the URL , so it is opening in a new Window , but there is no event on the newWindowRequested
This is the code I am using 
`WebView.CoreWebView2.NewWindowRequested += CoreWebView2_NewWindowRequested;`

So what I want is , I should get that event on the newWindowRequested event handler such that I can prevent it from opening in the new window.

cc : @JosephJin0815 , @champnic , @yild

### Importance

Blocking. My app's basic functions are not working due to this issue.

### Runtime Channel

Stable release (WebView2 Runtime)

### Framework

WPF

### Operating System

Windows 11

### Repro steps

Open URL in webview 
On right clicking , click on share 
It should raise an event on this event handler newWindowRequested.

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

No, this never worked

**Comment by champnic** [comment]
Thanks for the bug report @rahullli - I've added this bug to our internal tracker. It does seem like this should be triggering the New Window event

**Comment by jls28** [comment]
I don't understand the question,

when I click on the share menu item,

this open a special windows,
and I don't think this is a web page with an url,
so it cannot have a NewWindowRequested event.

(runtime last 129.xxxxxxxx).

**Comment by rahullli** [comment]
> I don't understand the question,
> 
> when I click on the share menu item,
> 
> this open a special windows, and I don't think this is a web page with an url, so it cannot have a NewWindowRequested event.
> 
> (runtime last 129.xxxxxxxx) / Win10.

So basically , I need to stop opening these special windows , as there is no event coming on the NewWindowRequested , I want to know from where exactly this window is opening , such that I can prevent it.
This Special window is being opened from the inspect and also from the share menu item .

**Comment by jls28** [comment]
what you can do is to remove the "Share" menu item, I believe it is possible.

**Comment by rahullli** [comment]
> what you can do is to remove the "Share" menu item, it is quite possible.

Yeah its one way , but I don't want to remove menu items which are opening these new window .

**Comment by rahullli** [comment]
> If you don't want remove the menu item I don't understand what you want
> 
> disable menu item ?
> 
> perform another thing if the user use this menu item ?

I want the user to use menu Items functionality such as Share and Inspect , but as they are being opening in a new window , so I want to open it in the same window , that's what I am looking for .

**Comment by jls28** [comment]
Ok, I don't think it is possible.

**Comment by rahullli** [comment]
Yeah , looking for the webview team to confirm if there exist any event handler for the same.

**Comment by rahullli** [comment]
Hii @champnic , any update on this bug ? 
If possible , can you provide some estimate , by when this can be fixed or any other way to handle this ?

**Comment by rahullli** [comment]
Hi @champnic
Any update on this Bug?
If possible , can you provide some estimate , by when this can be fixed or any other way to handle this ?
